test/utils/feature.flag.spec.ts

```diff
@@ -1,60 +1,32 @@
-import { describe, expect, test } from '@jest/globals';
+import { describe, expect } from '@jest/globals';
 import { isActiveFeature } from "../../src/utils/feature.flag";
 
-describe("feature flag tests", function() {
-
-  test("should return false if variable is 'false'", function() {
-    const active = isActiveFeature("false");
-    expect(active).toBeFalsy;
-  });
-
-  test("should return false if variable is '0'", function() {
-    const active = isActiveFeature("0");
-    expect(active).toBeFalsy;
-  });
-
-  test("should return false if variable is ''", function() {
-    const active = isActiveFeature("");
-    expect(active).toBeFalsy;
-  });
-
-  test("should return false if variable is undefined", function() {
-    const active = isActiveFeature(undefined);
+describe("feature flag tests", function () {
+  const dataExpectFalse = [
+    'false',
+    'faLse',
+    '0',
+    'off',
+    'oFF',
+    'kdjhskjf',
+  ];
+
+  test.only.each(dataExpectFalse)('should return false if variable is %p', (val) => {
```

Comment:
just reading up on the use of .only - https://stackoverflow.com/a/44446669
it looks like it is used to only run one specified test in a file, i guess when you have a large amount of tests you can run one in isolation. I think we can remove the .only from these tests.